app.py

```
@@ -1,420 +1,132 @@
-import os
-import json
-from threading import Thread
+from flask import Flask, render_template, request, jsonify
+from connection import Connection
+import threading
+import asyncio
+import logging
+from bleak import BleakScanner
 from flask import Response
+import queue
+import threading
```

Ruff (0.8.2)

Redefinition of unused `threading` from line 3

Remove definition: `threading`

(F811)